native/cocos/core/scene-graph/Node.cpp

```diff
@@ -520,9 +520,9 @@ Mat4 Node::getWorldRT() {
 void Node::invalidateChildren(TransformBit dirtyBit) { // NOLINT(misc-no-recursion)
     auto curDirtyBit{static_cast<uint32_t>(dirtyBit)};
     const uint32_t hasChangedFlags = getChangedFlags();
-    const uint32_t dirtyFlags = getDirtyFlag();
+    const uint32_t dirtyFlags = _transformFlags;
```

Comment:
The variable name `dirtyFlags` should be renamed to `transformFlags` either.

Reply:
fixed